### DISCUSSION TITLE
Go 1.18

### DISCUSSION THREAD

**Discussion by S-YOU** [99designs/gqlgen#2065] [Q&A] [answered]
### What happened?
Seems does not work with Go 1.18, because some files were not generated like `exec/filename`, `model/filename`
Probably the problem is with Go 1.18, but I just want to let you know.

### What did you expect?
Generate properly

### Minimal graphql.schema and models to reproduce
Private

### versions
 - `go run github.com/99designs/gqlgen version`?
     - only v0.17.1 and v0.17.2  is tried
 - `go version`?
     - Go 1.18 (No problem with Go1.17)

**Comment by frederikhors** [comment, marked as answer]
It works with go 1.18: I'm using it. Please create a reproduction repo.

**Reply by S-YOU** [reply]
I see. I can't post graphql schema because it is part of company private repo on product that is not release yet.
You may close this issue. Thank you for confirmation.
